tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+skipsdist = true
+envlist = tests, flake8
```

Comment:
I think we should run these tests in multiple Python versions, but that can be a later enhancement.